extras/spoof.md

```diff
@@ -47,6 +47,34 @@ With this, navigate towards `Root -> DeviceProperties -> Add` and add your PCI r
 
 ![](../images/extras/spoof-md/config-gpu.png)
 
+For disabling NVIDIA Kepler on per-slot-basis you can add these device properties. Find the path of the NVIDIA GPU with gfxutil:
+```
+/Users/(YourUsername)/Downloads/(gfxdownload folder)/gfxutil -f GFX0
+
+```
+```
+<key>DeviceProperties</key>
+<dict>
+<key>Add</key>
+<key>PciRoot(0x0)/Pci(0x7,0x0)/Pci(0x0,0x0)/Pci(0x8,0x0)/Pci(0x0,0x0)</key>
+<dict>
+<key>AAPL,ndrv-dev</key>
+<false/>
+</dict>
+</dict>
+
+```
+```
+<key>Delete</key>
```

Comment:
Which macs have this property defined in their firmware? Went through a few Kepler firmware dumps but found nothing

Reply:
No, this property is added to delete an existing property. In this case it is `ndrv-dev ` TRUE when NVIDIA card is initialized. Without it `ndrv-dev ` FALSE can not be set. 
https://github.com/acidanthera/OpenCorePkg/blob/master/Docs/Sample.plist#L291